.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: '🐛 Bug Report'
+about: Report a reproducible bug or regression in this library.
+---
+
+# Bug
+
+<!--
+  Please provide a clear and concise description of what the bug is.
+  Include screenshots if needed.
+  Please test using the latest release of the library, as maybe said bug has been already fixed.
+-->
+
+## Environment info
+
+<!--
+  Run `react-native info` in your terminal and copy the results here. And add the version of the library that you are using
```

Comment:
I would make it clearer that they need to add the library version number. That's one of the most important bits and is very likely to get missed if people just add the `react-native info` results.

Reply:
Uhm, how would you phrase `And add the version of the library that you are using` differently? 🤔

To, like, `Also, include the precise version number of this library that you are using in the project`?